bundles/org.openhab.binding.heliosventiremote/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,112 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="heliosventiremote"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Ventilation Thing Type -->
+	<thing-type id="heliosVentilation">
+		<label>HeliosVentilation (KWL)</label>
+		<description>A domestic ventilation system (KWL) from Helios.</description>
+
+		<channels>
+            <channel id="outsideTemp" typeId="outside_temperature" />
+            <channel id="outgoingTemp" typeId="outgoing_temperature" />
+            <channel id="extractTemp" typeId="extract_temperature" />
+            <channel id="supplyTemp" typeId="supply_temperature" />
+            <channel id="bypass_temp" typeId="bypass_temperature" />
+            <channel id="min_fanspeed" typeId="min_fanspeed" />
+            <channel id="max_fanspeed" typeId="max_fanspeed" />
+            <channel id="fanspeed" typeId="fanspeed" />
+            <channel id="rh_limit" typeId="rh_limit" />
+		</channels>
+
+        <config-description>
+            <parameter name="serialPort" type="text" required="true">
+                <context>serial-port</context>
+                <label>RS485 Interface Serial Port</label>
+                <description>The serial port name for the RS485 interfaces. Valid values
+                    are e.g. COM1 for Windows and /dev/ttyS0 or
+                    /dev/ttyUSB0 for Linux.</description>
+            </parameter>
+
+            <parameter name="pollPeriod" type="integer" required="true" min="0">
+                <label>Poll Period (sec.)</label>
+                <description>The Poll Period in Seconds use 0 for no polling.</description>
+                <default>60</default>
+            </parameter>
+        </config-description>
+	</thing-type>
+
+    <channel-type id="outside_temperature" advanced="false">
+        <item-type>Number</item-type>
```

Comment:
We have unit-support now, please use `Number:Temperature` for temperature-channels (also applies to other temperature channels)